internal/config/config.go

```diff
@@ -290,6 +290,11 @@ func setClowderConfiguration(v *viper.Viper, clowderConfig *clowder.AppConfig) {
 		v.Set("logging.cloudwatch.group", clowderConfig.Logging.Cloudwatch.LogGroup)
 		v.Set("logging.cloudwatch.region", clowderConfig.Logging.Cloudwatch.Region)
 		v.Set("logging.cloudwatch.secret", clowderConfig.Logging.Cloudwatch.SecretAccessKey)
+		// TODO Delete the block below when the below PR is merged
+		// See: https://github.com/RedHatInsights/clowder/pull/627
+		if clowderConfig.Logging.Type == "" {
```

Comment:
Maybe also check if `len(clowderConfig.Logging.Cloudwatch.SecretAccessKey) > 0`?

Reply:
How about we only check for secret access keys? That way the code will still work when https://github.com/RedHatInsights/clowder/pull/627 has been merged.

Reply:
IMHO I don't know if another logging providers will coexists at some moment in the future; the guard `Type == ""` avoid to overwrite if some value exists at that moment, the check `len(clowderConfig.Logging.Cloudwatch.SecretAccessKey) > 0` will additionally set the value when we have some information for cloudwatch.

The current change accomplish:
- avoid overwrite value if some content exists.
- avoid to set the value if maybe cloudwatch object exists but the contents are empty.
- this let to set `"cloudwatch"` value meanwhile the PR is merged and will let the code works after merging it.

Reply:
I think the workaround is good as it is right now. In future you might have cloudwatch secrets but logging.type configured to something else and want to react on this.